modules/core/src/v2/wallet.ts

```diff
@@ -424,6 +426,7 @@ export interface SendManyOptions extends PrebuildAndSignTransactionOptions {
   instant?: boolean;
   memo?: Memo;
   transferId?: number;
+  intentType?: string;
```

Comment:
don't think we should expose this field yet. im not 100% convinced we want backwards compatibility with `send` and `sendMany` now that we're moving towards TxRequests / Intents.

to start, let's assume that all TSS sends are payments until we figure out how to proceed